easybuild/easyconfigs/l/libxml2/libxml2-2.9.14-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,27 @@
+name = 'libxml2'
```

Comment:
I don't see this being listed as a dependency of PAPI, so I don't really understand why this easyconfig needs to be added here?

Reply:
It is a dependency of Extrae (same as PAPI). It is not needed here; it just happened that I made the commit and pushed it to my fork before the PR was solved, and this is why it appears here.

Reply:
Okay, in that case, shall we remove it from this PR?

Reply:
Yes, probably it makes sense to remove it from this PR and make a separated one for it